src/main/app/drafts/models/defendant.ts

```diff
@@ -1,11 +1,14 @@
 import { Address } from 'forms/models/address'
+import { DefendantDetails } from 'app/forms/models/defendantDetails'
 
 export default class Defendant {
   address?: Address = new Address()
+  personalDetails?: DefendantDetails = new DefendantDetails()
```

Comment:
Just a thought, personalDetails doesn't look appropriate name here. s it contains company house no and organisation. You may call it `defendantDetails` or think something better.